src/main/resources/tags/links/uuid.tag

```diff
@@ -2,5 +2,16 @@ type: text
 aliases: xuid, floodgateuuid, bedrockuuid
 
 ---
+Automatic method:
+Use https://uuid.kejona.dev/ to convert any Bedrock Player username into a Floodgate UUID.
 
-You can find a Bedrock player's UUID before they join using this (for whitelists for example): https://floodgate-uuid.heathmitchell1.repl.co/
+Manual method:
+Unfortunately, the floodgateuuid converter currently does not work. In the meantime, you can manually find out a UUID using this method:
+Step 1: Visit https://www.cxkes.me/xbox/xuid and enter the bedrock username (without the period that is added by Geyser)
+Step 2: Search for the hexidecimal result and copy it down
+Step 3: You can try just that number instead of the username, but mine was formatted like this
+
+XUID Search Result: 000901F4BF5B7415
+Minecraft XUID: 00000000-0000-0000-0009-01f4bf5b7415
+
+So step 4 would be to add a dash after the first four digits and then paste "00000000-0000-0000-" before the search result to get the correct format.
```

Comment:
Honestly I would remove all this. There are two websites currently online for it and if they ever both go offline we can just pin a message in the support channel in the meantime

Reply:
right, I'll include a proper link to the thread